pmd-ui/pom.xml

```diff
@@ -26,5 +42,50 @@
             <artifactId>pmd-java</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.fxmisc.richtext</groupId>
+            <artifactId>richtextfx</artifactId>
+            <version>0.8.1</version>
+        </dependency>
+        <dependency>
+            <groupId>net.sourceforge.pmd</groupId>
+            <artifactId>pmd-apex</artifactId>
+            <version>6.0.0-SNAPSHOT</version>
```

Comment:
Please use "${project.version}" for the pmd-owned dependencies - that way, we'll have less trouble for releasing.